entrypoint.sh

```
@@ -457,4 +457,188 @@ echo "   Total time: $((UPLOAD_DURATION + CONFIRM_DURATION)) seconds"
+  sleep "$POLL_INTERVAL"
+done
+
+# Final summary
+echo ""
+echo "========================================"
+echo "  Results Summary"
+echo "========================================"
+
+TOTAL_GROUPS=$(echo "$STATUS_RESPONSE" | jq -r '.summary.total_groups')
+PASSED_GROUPS=$(echo "$STATUS_RESPONSE" | jq -r '.summary.passed_groups')
+FAILED_GROUPS=$(echo "$STATUS_RESPONSE" | jq -r '.summary.failed_groups')
```

Shellcheck (0.11.0)

[warning] FAILED_GROUPS appears unused. Verify use (or export if used externally).

(SC2034)